ansible/roles/podman/tasks/config.yml

```diff
@@ -15,3 +15,60 @@
 
 - name: reset ssh connection to allow user changes to affect 'current login user'
   meta: reset_connection
+
+- name: Ensure podman users exist
+  user: "{{ item }}"
+  with_items: "{{ podman_users }}"
+  register: podman_user_info
+
+- name: Define tmp directories on tmpfs
+  blockinfile:
+    path: /etc/tmpfiles.d/podman.conf
+    create: yes
+    block: |
+      d {{ podman_tmp_dir_root }}/{{ item.name }}/libpod/tmp 0755 podman podman
```

Comment:
Have we lost the change that got rid of the podman hard-coding here?

Reply:
Sorry don't understand the comment?

Reply:
> Sorry don't understand the comment?

Sorry, was referring to this change:

https://github.com/stackhpc/openhpc-demo/pull/52/commits/85a12dee4115be4a6fa3cefc680950af325fc9ba